automatic/wixtoolset/wixtoolset.nuspec

```
@@ -11,11 +11,11 @@
     <projectUrl>http://wixtoolset.org/</projectUrl>
     <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@bfbac575d697b67c7930af094980146278045677/icons/wixtoolset.png</iconUrl>
     <copyright>2009 - 2016 Outercurve Foundation</copyright>
-    <licenseUrl>http://wixtoolset.org/about/license/</licenseUrl>
+    <licenseUrl>https://raw.githubusercontent.com/wixtoolset/wix3/develop/LICENSE.TXT</licenseUrl>
```

Do **NOT** use raw GitHub URLs to the content, instead use the link to the file when viewing the repository.

```suggestion
    <licenseUrl>https://github.com/wixtoolset/wix3/blob/develop/LICENSE.TXT</licenseUrl>
```

---

That does not make sense at all!
This is a TXT file and therefore, the raw URL was taken intentionally (showing ONLY the license).
Otherwise legal people might be confused by all the github GUI elements around it.